### ISSUE TITLE
`byte-array` is undeclared

### ISSUE THREAD

**Issue by chrismurrph** [replikativ/hasch#16]
Use of undeclared Var hasch.core/byte-array
  55 |   all bits of the hash compared to 128 bits for the UUID-5. Both should be safe,
  56 |   but b64-hash is safer towards collisions."
  57 |   [val]
  58 |   (b64/encode (byte-array (edn-hash val))))
  59 | 
  60 | 

Is this because `byte-array` only exists in Clojure and this is a .cljc file?

**Comment by chrismurrph** [comment]
I know this is fixed in latest source, so this issue in the wrong place. But that error message keeps popping up for me.
   io.replikativ/konserve     {:mvn/version "0.5.1"}
   io.replikativ/hasch        {:mvn/version "0.3.7"}
Perhaps "0.5.1" somehow uses an older version?? I'm thinking that having now put "0.3.7" in things should be okay.

**Comment by whilo** [comment]
Yes, you can manually bump the hasch version. We are also currently working to a 0.6.0 release of konserve which also has this fixed.
